crates/simulation/src/main.rs

```diff
@@ -747,7 +746,7 @@ fn create_pipelines(device: &wgpu::Device, surface_format: wgpu::TextureFormat,
 }
 
 fn surface_texture_format() -> wgpu::TextureFormat {
-    wgpu::TextureFormat::Rgba8UnormSrgb
+    wgpu::TextureFormat::Bgra8UnormSrgb
```

Comment:
This change to `Bgra8UnormSrgb` works fine on my end. 
But the right format is hardware independent and it is probably a better idea to get a list of supported formats and just go with the first one, like `supported_formats.first()`